cmd/dlt/ocmrole/cmd.go

```diff
@@ -219,36 +236,38 @@ func buildCommands(roleName string, roleARN string, isLinked bool, awsClient aws
 		commands = append(commands, unlinkRole)
 	}
 
-	policies, err := awsClient.GetAttachedPolicy(&roleName)
-	if err != nil {
-		return "", err
-	}
-	for _, policy := range policies {
-		detachPolicy := fmt.Sprintf("aws iam detach-role-policy \\\n"+
-			"\t--role-name %s \\\n"+
-			"\t--policy-arn %s",
-			roleName, policy.PolicyArn)
-		commands = append(commands, detachPolicy)
-		deletePolicy := fmt.Sprintf("aws iam delete-policy \\\n"+
-			"\t--policy-arn %s",
-			policy.PolicyArn)
-		commands = append(commands, deletePolicy)
-	}
+	if roleExistOnAWS {
+		policies, err := awsClient.GetAttachedPolicy(&roleName)
+		if err != nil {
+			return "", err
+		}
+		for _, policy := range policies {
+			detachPolicy := fmt.Sprintf("aws iam detach-role-policy \\\n"+
+				"\t--role-name %s \\\n"+
+				"\t--policy-arn %s",
+				roleName, policy.PolicyArn)
+			commands = append(commands, detachPolicy)
+			deletePolicy := fmt.Sprintf("aws iam delete-policy \\\n"+
+				"\t--policy-arn %s",
+				policy.PolicyArn)
+			commands = append(commands, deletePolicy)
+		}
 
-	hasPermissionBoundary, err := awsClient.HasPermissionsBoundary(roleName)
-	if err != nil {
-		return "", err
-	}
-	if hasPermissionBoundary {
-		deletePermissionBoundary := fmt.Sprintf("aws iam delete-role-permissions-boundary \\\n"+
-			"\t--role-name %s",
-			roleName)
-		commands = append(commands, deletePermissionBoundary)
-	}
+		hasPermissionBoundary, err := awsClient.HasPermissionsBoundary(roleName)
+		if err != nil {
+			return "", err
+		}
+		if hasPermissionBoundary {
+			deletePermissionBoundary := fmt.Sprintf("aws iam delete-role-permissions-boundary \\\n"+
+				"\t--role-name %s",
+				roleName)
+			commands = append(commands, deletePermissionBoundary)
+		}
```

Comment:
I think the indentation here is incorrect.

Reply:
Seems like some error on Git:
![image](https://user-images.githubusercontent.com/57869309/162624483-73dddc82-0651-4252-a831-c74945a6326c.png)